next/components/forms/simple-components/ServiceCard.tsx

```diff
@@ -0,0 +1,49 @@
+import cx from 'classnames'
+import { FunctionComponent, HTMLAttributeAnchorTarget, SVGProps } from 'react'
+
+import CircleArrowRight from '../../../assets/images/circle-arrow-right.svg'
+
+type ServiceCardBase = {
+  title: string
+  description: string
+  buttonText: string
+  Logo: FunctionComponent<SVGProps<SVGSVGElement>>
+  href: string
+  target?: HTMLAttributeAnchorTarget | undefined
+  className?: string
+}
+
+const ServiceCard = ({
+  title,
+  description,
+  buttonText,
+  Logo,
+  href,
+  target,
+  className,
+}: ServiceCardBase) => {
+  const style = cx(
+    'w-[280px] bg-gray-0 border-gray-200 flex flex-col items-start p-5 gap-5 border-solid border-2 rounded-lg',
+    className,
+  )
+
+  return (
+    <a href={href} target={target} rel="noreferrer">
+      <div role="button" tabIndex={0} className={style} aria-hidden="true">
```

Comment:
why would you do `aria-hidden="true"` ? this will hide the whole subcomponent from the screenreaders. Also why `tabIndex={0}` ? a static value does not make sense here, you do not know where these cards will be used.

Reply:
also I don't think you need role="button", that will be handled by the wrapping <Link> or <a> (as mentioned in the other comment, outside of the visual component) - was this just forgotten ?